### PR TITLE
More commands added.

### PR DESCRIPTION
**Pull request by Hodobox**

### CODE REVIEW

**Review by kubik369 — APPROVED** [review]

File: src/main/java/graphgui/GraphPane.java [hunk 1]
```diff
@@ -22,7 +25,7 @@
 /** Trieda pre grafický panel, na ktorom umiestňujeme nakreslenie grafu. */
 public class GraphPane extends Pane implements ExtendedGraph.GraphObserver {
   private static final int EDGE_WIDTH = 4;
-  
+
```
Comment (Siegrift) [inline src/main/java/graphgui/GraphPane.java:28]: Preco su tu tieto typy zmien?

File: src/main/java/graphgui/GraphPane.java [hunk 2]
```diff
@@ -315,8 +318,14 @@ public void executeCommand(String[] tokens) {
           break;
         }
 
-        //TODO: zavolat funkciu 'Save' ktora je v menu, nazov suboru = tokens[1]
-        if (true) { // ak sa podarilo
+        String filename = new String(tokens[1]);
+        if (filename.endsWith(".graph") == false) {
+          filename = new String(filename + ".graph");
+        }
+
```
Comment (Siegrift) [inline src/main/java/graphgui/GraphPane.java:325]: Unnecessary newlines.

File: src/main/java/graphgui/GraphPane.java [hunk 3]
```diff
@@ -332,8 +341,14 @@ public void executeCommand(String[] tokens) {
           break;
         }
 
-        //TODO: zavolat funkciu 'Load' ktora je v menu, nazov suboru = tokens[1]
-        if (true) { // ak sa podarilo
+        String filename = new String(tokens[1]);
+        if (filename.endsWith(".graph") == false) {
+          filename = new String(filename + ".graph");
+        }
+
```
Comment (Siegrift) [inline src/main/java/graphgui/GraphPane.java:346]: Unnecessary newlines.

**Review by Siegrift — CHANGES_REQUESTED** [review]
Nice, but try not to add additional newLines if not neccessary. Put comment if you want to emphesize on something. 

And we should probably think about splitting the code as its getting too long. Probably starting with moving command line commands to separate class.

**Review by Siegrift — APPROVED** [review]

**Pull request merged**